src/index.ts

```diff
@@ -47,6 +47,17 @@ function createQuery(parameters: object): string {
     .replace(/^&/, '?');
 }
 
+function filterFormData(args: any[], parameters?: FormDataParameter[]): any[] {
+  const urlParams: any[] = [];
+  args.reduce((memo: any[], arg, index) => {
```

Comment:
If you already call the function 'filter` why not use the `filter` function instead of  `reduce`?